setup/requirements.md

```diff
@@ -53,6 +53,29 @@ currently require the following versions _or newer_:
 If you're using [Remote IDEs](../workspaces/editors.md), allow pop-ups; Coder
 launches the Remote IDE in a pop-up window.
 
+## Storage
+
+Coder requires the use of a [persistent
+volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) in your
+Kubernetes cluster to store [workspaces](../workspaces/index.md) data. More
+specifically, the persistent volume claim (PVC) requires the block storage type
+(the PVC is created when you create the workspace to mount the requested block
+storage).
```

Comment:
@ericpaulsen sorry to reopen an old discussion I must have missed, but I remember seeing this in our community slack regarding a bare-metal install:

> It looks like you're missing a dynamic volume provisioner, which I can imagine happening easily if you're running "on bare metal". See https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/ for what that's about, but see https://kubernetes.io/docs/concepts/storage/storage-classes/ for the main documentation on how to fix it. You'll likely want to set a "default" storage class (see https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/), unless you have particular needs, whereupon you'll need to tweak the Coder helm chart to set the storage class on the generated PVCs. (edited) 

Asking because I'm genuinely not sure - does this description infer the use of a Dynamic Volume Provisioner?

Reply:
@bpmct I _think_ Coder assumes the use of Dynamic Volume Provisioning (per the definition), but I'm not entirely sure. I'm waiting on some feedback from product to validate. I'll be sure to update docs accordingly.

Reply:
Can confirm that Coder assumes the use of Dynamic Volume Provisioning to properly mount PVCs & create workspaces. I'll note this in a new PR.